src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCITriggerService.java

```diff
@@ -53,6 +55,16 @@
 @Profile(PROFILE_LOCALCI)
 public class LocalCITriggerService implements ContinuousIntegrationTriggerService {
 
+    public static final int PRIORITY_LOWEST = 4;
+
+    public static final int PRIORITY_LOW = 3;
+
+    public static final int PRIORITY_BASE = 2;
+
+    public static final int PRIORITY_HIGH = 1;
+
+    public static final int PRIORITY_PENALTY = 5;
```

Comment:
I don't really like the names, since they don't add anything compared to numbers and actually make it harder, e.g. how many priorities are before PRIORITY_LOW?
I suggest using names like NORMAL, EXAM_CONDUCTION, EXAM_PREPARATION, UNRATED_EXERCISE and TRIGGER_ALL. This way one can easily see the assigned priorities here in the constants and we can also easily change the priority of a specific category. Also nothing would stop us from assigning the same priority to different groups